src/nci_util.c

```diff
@@ -402,6 +426,69 @@ nci_parse_iso_dep_poll_a_param(
     return FALSE;
 }
 
+static
+gboolean
+nci_parse_iso_dep_poll_b_param(
+    NciActivationParamIsoDepPollB* param,
+    const guint8* bytes,
+    guint len)
+{
+    /* NFCFrum-TS-NCI-1.0
+     * Table 75: Activation parameters for NFC-B/ISO-DEP Poll Mode
+     *
+     * +============================================================+
+     * | Offset | Size | Description                                |
+     * +============================================================+
+     * | 0      | 1    | Length of ATTRIB Response Parameter (n)    |
+     * | 1      | n    | ATTRIB Response                            |
+     * +============================================================+
+     */
+
+    /* ATTRIB Response */
+    const guint attrib_length = bytes[0];
+
+    if (attrib_length >= 1) {
+        /* NFCForum-TS-DigitalProtocol-1.01
+         * Table 79: ATTRIB Response Format */
+
+#define NFC_T4B_MBLI_MASK (0xF0) /* MBLI Mask */
+#define NFC_T4B_DID_MASK  (0x0F) /* DID Mask */
+
+        /* First two octets of the first byte of ATTRIB Response */
+        param->did  = (bytes[1] & NFC_T4B_DID_MASK);
+        param->mbli = (bytes[1] & NFC_T4B_MBLI_MASK) >> 4;
```

Comment:
Please swap either these two statements (or the defines). I suggest to follow the order used in the spec (MBLI followed by DID). That makes it easier to read.